### ISSUE TITLE
NET0470 Table 1 Requirements not tested in astm.netrid.v22a.NominalBehavior

### ISSUE THREAD

**Issue by wing-utm-sharing-airspace** [interuss/monitoring#857]
**Observed behavior**
In scenarios.astm.netrid.v22a.NominalBehavior, NET0407 requirements are not being verified during the "Observer polling" step.

We're currently interested in:
NET0470,Table1,1
NET0470,Table1,1a
NET0470,Table1,9
NET0470,Table1,19
NET0470,Table1,20

Example Test Artifacts: [aac5c156-dc1c-4ef4-934a-64c50ea5226f.zip](https://github.com/user-attachments/files/18000759/aac5c156-dc1c-4ef4-934a-64c50ea5226f.zip)

**Test check**
Screenshot of our USS populating the relevant fields:
![image](https://github.com/user-attachments/assets/814baf1c-3339-4299-8504-f219a35a9b4c)
Scenario 31 Step 759 in attached test artifacts.

**Difference from expected behavior**
Our USS is populating the UAS ID, Operator ID, speed, and track when our observer is polled so we believe the NET0407 requirements in the `phase1_requirements/wing.html` file should be marked as "passed" where this test report shows them as "not tested".
![Screenshot 2024-12-04 at 10 53 43 AM](https://github.com/user-attachments/assets/26030991-f938-46f0-aff9-4982666d5473)

**Comment by BenjaminPelletier** [comment]
@barroco or @mickmis, is this perhaps because we're [looking for uas_id](https://github.com/interuss/monitoring/blob/58d636e06c5f05011bc668aeee85fc5d4e439694/monitoring/uss_qualifier/scenarios/astm/netrid/common_dictionary_evaluator.py#L210) rather than uas.id?

**Comment by mickmis** [comment]
@BenjaminPelletier indeed. More generally, those checks are reused for details coming both from the automated testing interface:
https://github.com/interuss/automated_testing_interfaces/blob/main/rid/v1/observation.yaml#L86-L95

and from the service provider interface: 
https://github.com/uastech/standards/blob/dd4016b09fc8cb98f30c2a17b5a088fb2995ab54/remoteid/updated.yaml#L1694-L1732

Yet those interfaces are not specified the same.

We were already supposed to do a pass on those parts of the code as part of the ongoing RID effort. I will make sure we include fixing this issue along the way.

**Comment by BenjaminPelletier** [comment]
It looks like we [currently](https://interuss.github.io/monitoring/artifacts/uss_qualifier/reports/uspace/requirements/uss1.html) appear not to have a check for NET0470,Table1,1a, and it looks like NET0470,Table1,1 is not included in our U-space dev configuration.  I think I would expect these two requirements to look a lot like NET0260,Table1,1 and NET0260,Table1,1a.

**Comment by the-glu** [comment]
> It looks like we [currently](https://interuss.github.io/monitoring/artifacts/uss_qualifier/reports/uspace/requirements/uss1.html) appear not to have a check for NET0470,Table1,1a, and it looks like NET0470,Table1,1 is not included in our U-space dev configuration.

I have this PR from last week (but I forgot to submit) that should do the trick for the `NET0470,Table1,1` part : https://github.com/interuss/monitoring/pull/1045

For the `NET0470,Table1,1a` it's trickier: since we don't know on the SP side witch UAS ID Type is sent, the current generic test suite is skipping sub tests ( `NET0470,Table1,1[a-d]`) if it doesn't match. If I understood correctly, we want to the U-space dev test to enforce `NET0260,Table1,1a`, that will need adaptions to optionally enforce that :)

**Comment by mickmis** [comment]
> For the `NET0470,Table1,1a` it's trickier: since we don't know on the SP side witch UAS ID Type is sent, the current generic test suite is skipping sub tests ( `NET0470,Table1,1[a-d]`) if it doesn't match. 

I actually cannot find any scenario checking for `NET0470,Table1,1[a-d]`. So we are at least missing that, I will prepare a PR to address that.

> If I understood correctly, we want to the U-space dev test to enforce `NET0260,Table1,1a`, that will need adaptions to optionally enforce that :)

Would it make sense for this to be added as a 'verifiable capability'? @BenjaminPelletier
